convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -853,6 +853,19 @@ def test_list_third_party_pkgs(self):
         self.assertEqual(len(pkghandler.print_pkg_info.pkgs), 3)
         self.assertTrue("Only packages signed by" in logger.CustomLogger.warning.msg)
 
+    @unit_tests.mock(tool_opts, "disablerepo", ['*', 'rhel-7-extras-rpm'])
+    @unit_tests.mock(tool_opts, "enablerepo", ['rhel-7-extras-rpm'])
+    @unit_tests.mock(logger.CustomLogger, "warning", LogMocked())
+    def test_is_disable_and_enable_repos_has_same_repo(self):
+        pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options()
+        self.assertTrue("Duplicate repositories was found" in logger.CustomLogger.warning.msg)
+
+
+    @unit_tests.mock(tool_opts, "disablerepo", ['*'])
+    @unit_tests.mock(tool_opts, "enablerepo", ['rhel-7-extras-rpm'])
+    def test_is_disable_and_enable_repos_doesnt_thas_same_repo(self):
+        self.assertEqual(None, pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options())
```

Comment:
This will also pass if there are duplicate repos. Add a mock for CustomLogger and assert that it is not called
```suggestion
    @unit_tests.mock(logger.CustomLogger, "warning", LogMocked())
    def test_is_disable_and_enable_repos_doesnt_thas_same_repo(self):
        pkghandler.has_duplicate_repos_across_disablerepo_enablerepo_options()
        self.assertEqual(logger.CustomLogger.warning.called, 0)
```